tests/chart_tests/test_airflow_operator.py

```diff
@@ -0,0 +1,123 @@
+        docs = render_chart(
+            validate_objects=False,
+            kube_version=kube_version,
+            values={
+                "airflow-operator": {
+                    "certManager": {"enabled": True},
+                },
+                "global": {
+                    "airflowOperator": {"enabled": True},
+                },
+            },
+            show_only=[
+                "charts/airflow-operator/templates/certmanager/selfsigned-issuer.yaml",
+                "charts/airflow-operator/templates/certmanager/serving-cert-certificate.yaml",
+            ],
+        )
+        assert len(docs) == 2
+        assert "Issuer" == docs[0]["kind"]
+        assert "Certificate" == docs[1]["kind"]
+        assert "cert-manager.io/v1" == docs[0]["apiVersion"]
+        assert "cert-manager.io/v1" == docs[1]["apiVersion"]
+        assert "release-name-airflow-operator-serving-cert" == docs[1]["metadata"]["name"]
+        assert "release-name-airflow-operator-selfsigned-issuer" == docs[0]["metadata"]["name"]
+
+    def test_airflow_operator_crd(self, kube_version):
+        """Test Airflow Operator crd template"""
+        docs = render_chart(
+            validate_objects=False,
+            kube_version=kube_version,
+            values={
+                "airflow-operator": {
+                    "crd": {"create": True},
+                },
+                "global": {
+                    "airflowOperator": {"enabled": True},
+                },
+            },
+            show_only=[
+                "charts/airflow-operator/templates/crds/airflow.yaml",
+                "charts/airflow-operator/templates/crds/allocator.yaml",
+                "charts/airflow-operator/templates/crds/apiserver.yaml",
+                "charts/airflow-operator/templates/crds/dagprocessor.yaml",
+                "charts/airflow-operator/templates/crds/pgbouncer.yaml",
+                "charts/airflow-operator/templates/crds/postgres.yaml",
+                "charts/airflow-operator/templates/crds/rbac.yaml",
+                "charts/airflow-operator/templates/crds/redis.yaml",
+                "charts/airflow-operator/templates/crds/runner.yaml",
+                "charts/airflow-operator/templates/crds/scheduler.yaml",
+                "charts/airflow-operator/templates/crds/statsd.yaml",
+                "charts/airflow-operator/templates/crds/triggerer.yaml",
+                "charts/airflow-operator/templates/crds/webserver.yaml",
+                "charts/airflow-operator/templates/crds/worker.yaml",
+            ],
+        )
+        assert len(docs) == 14
+        for i in range(len(docs)):
+            assert "apiextensions.k8s.io/v1" == docs[i]["apiVersion"]
+            assert "CustomResourceDefinition" == docs[i]["kind"]
+            assert "cert-manager.io/inject-ca-from" in docs[i]["metadata"]["annotations"]
+            assert "airflow.apache.org" in docs[i]["metadata"]["name"]
```

Comment:
Shorter, cleaner syntax
```suggestion
        for doc in docs:
            assert "apiextensions.k8s.io/v1" == doc["apiVersion"]
            assert "CustomResourceDefinition" == doc["kind"]
            assert "cert-manager.io/inject-ca-from" in doc["metadata"]["annotations"]
            assert "airflow.apache.org" in doc["metadata"]["name"]
```